templates/frontend/objects/issue_toc.tpl

```diff
@@ -17,109 +17,92 @@
-							<div class="description_text">
-								{$issueDescription|substr:0:800|mb_convert_encoding:'UTF-8'|replace:'?':''}<span
-										class="three_dots">...</span>
-								<a class="more_button"
-								   href="{url op="view" page="issue" path=$issue->getBestIssueId()}">
-									{translate key="plugins.themes.classic.more"}
-								</a>
-							</div>
-						{else}
-							<div class="description_text">
-								{$issueDescription}
-							</div>
-						{/if}
+{* Issue introduction area above articles *}
+{if $issue->hasDescription() || $issue->getLocalizedCoverImageUrl()}
+	{* Description *}
+	<section class="flex_container description_cover">
+		{if $issue->hasDescription()}
+			<div class="description">
+				<h3 class="description_label">{translate key="plugins.themes.classic.issueDescription"}</h3>
+				{assign var=issueDescription value=$issue->getLocalizedDescription()|strip_unsafe_html}
+				{if $issueDescription|strlen < 800}
+					<div class="description_text">
+						{$issueDescription}
 					</div>
-				{/if}
-
-				{* Issue cover image *}
-				{assign var=issueCover value=$issue->getLocalizedCoverImageUrl()}
-				{if $issueCover}
-					<div class="issue_cover_block{if !$issue->hasDescription()} align-left{/if}">
-						<a href="{url op="view" page="issue" path=$issue->getBestIssueId()}">
-							<img class="cover_image"
-							     src="{$issueCover|escape}"{if $issue->getLocalizedCoverImageAltText() != ''} alt="{$issue->getLocalizedCoverImageAltText()|escape}"{/if}>
+				{elseif $requestedPage|escape !== "issue"}
+					<div class="description_text">
+						{$issueDescription|substr:0:800|mb_convert_encoding:'UTF-8'|replace:'?':''}<span
+								class="three_dots">...</span>
+						<a class="more_button"
+						   href="{url op="view" page="issue" path=$issue->getBestIssueId()}">
+							{translate key="plugins.themes.classic.more"}
 						</a>
 					</div>
+				{else}
+					<div class="description_text">
+						{$issueDescription}
+					</div>
 				{/if}
 			</div>
-		</div>
-	{/if}
+		{/if}
 
-	{* Full-issue galleys *}
-	{if $issueGalleys}
-		<div class="galleys">
-			<h2 class="sr-only">
-				{translate key="issue.tableOfContents"}
-			</h2>
-			<ul class="galleys_links">
-				{foreach from=$issueGalleys item=galley}
-					<li>
-						{include file="frontend/objects/galley_link.tpl" parent=$issue purchaseFee=$currentJournal->getSetting('purchaseIssueFee') purchaseCurrency=$currentJournal->getSetting('currency')}
-					</li>
-				{/foreach}
-			</ul>
-		</div>
-	{/if}
+		{* Issue cover image *}
+		{assign var=issueCover value=$issue->getLocalizedCoverImageUrl()}
+		{if $issueCover}
+			<div class="issue_cover_block{if !$issue->hasDescription()} align-left{/if}">
+				<a href="{url op="view" page="issue" path=$issue->getBestIssueId()}">
+					<img class="cover_image"
+					     src="{$issueCover|escape}"{if $issue->getLocalizedCoverImageAltText() != ''} alt="{$issue->getLocalizedCoverImageAltText()|escape}"{/if}>
```

Comment:
Just FYI, I learned this recently about setting `alt` tags to `null`: https://github.com/pkp/pkp-lib/issues/4557